components/brave_wallet/common/encoding_utils.cc

```diff
@@ -5,11 +5,28 @@
 
 #include "brave/components/brave_wallet/common/encoding_utils.h"
 
+#include <utility>
+
+#include "base/containers/span.h"
+#include "base/containers/span_writer.h"
 #include "brave/components/brave_wallet/common/hash_utils.h"
 #include "brave/third_party/bitcoin-core/src/src/base58.h"
 
 namespace brave_wallet {
 
+namespace {
+// Prefix is added a public key to calculate
+// blake2b hash.
+constexpr char kSs58HashPrefix[] = "SS58PRE";
+constexpr size_t kSs58HashPrefixSize = 7u;
```

Comment:
Don't need `kSs58HashPrefixSize` constant.
Something like `auto prefix = base::byte_span_from_cstring(kSs58HashPrefix);`
and `prefix.size()`
as a replacement